### PR TITLE
Spectra v2.0.12

### PR DESCRIPTION
**Pull request by RajkiranBagal**
* Improvement: How to block - Option to disable step links.
* Improvement: Forms - Added 'Required' toggle for Accept field/sub-block.
* Improvement: Content Timeline - Changed default description.
* Improvement: Media Picker - Updated Video Icon.
* Improvement: Post Carousel - Gap between Content & Dots functionality.
* Fix: Price list - Heading are causing overlap on mobile on front-end.
* Fix: Forms Toggle - Changed default label from 'Name' to 'Toggle'.
* Fix: Forms - Fixed alignment for required toggle button in Phone block and added spacing between toggle and the 'Required' label.
* Fix: Content Timeline - Removed extra margin added to title and description by theme and core WordPress in editor and frontend.
* Fix: Info Box - Images looked enlarged after upgrading to Spectra.
* Fix: Form - Form is sending empty data after adding keys for ReCaptcha V3 .
* Fix: Forms - Required Toggle for Checkbox field was not clickable in editor.
* Fix: Advanced Row - Border not visible for backward user.
* Fix: Price List - Changed default padding for images so that the image aligns with the title.
* Fix: Responsive Border - Border width occasionally misbehaves when set to zero.
* Fix: Price List - Columns settings are not working in mobile editor.
* Fix: Container - Shape divider background color issue on editor.
* Fix: Responsive condition - Setting does not save when hide on multiple devices is selected.
* Fix: How-To - Alignment for time and cost not working in editor as well as on frontend.
* Fix: Post Carousel and Masonry - Background Image overlay color is not same as frontend.
* Fix: Advanced Row - Border hover color not working on front end.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,29 @@ When you use the Spectra along with the free Astra theme, you get a huge library
 
 ## Changelog ##
 
+### 2.0.12 - THURSDAY, 15th SEPTEMBER 2022 ###
```
Comment (AbhijitNage123) [inline README.md:169]: ### 2.0.12 - FRIDAY, 16th SEPTEMBER 2022 ###

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]

**Review by supriyajjadhav — APPROVED** [review]

File: src/blocks/extensions/condition-block/index.js [hunk 2]
```diff
@@ -127,23 +128,23 @@ const UserResponsiveConditionOptions = ( props ) => {
...
+						UAGResponsiveConditions: true,
 					} )}
 				/>
 				<ToggleControl
 					label={ __( 'Hide on Tablet' ) }
 					checked={UAGHideTab}
                     onChange={() => setAttributes( {
 						UAGHideTab: ! attributes.UAGHideTab,
-						UAGResponsiveConditions: ! UAGResponsiveConditions
+						UAGResponsiveConditions: true,
```
Comment (aapatre) [inline src/blocks/extensions/condition-block/index.js:139]: Why are we always setting the value to `true` for onChange event of a ToggleControl? :thinking:

File: README.md (hunk 1 above)
Comment (AbhijitNage123) [reply README.md:169]: PR - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/2313

File: src/blocks/extensions/condition-block/index.js (hunk 2 above)
Comment (AbhijitNage123) [reply src/blocks/extensions/condition-block/index.js:139]: It is intentional, UAGResponsiveConditions should be true on the onChange event. In this way, we are able to set multiple responsive conditions for a block. It does not impact anything.

Comment (aapatre) [reply src/blocks/extensions/condition-block/index.js:139]: Alright, marking this as resolved

**Review by aapatre — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

**Review by jamyrbsf — APPROVED** [review]

**Pull request merged**